app/schemas.py

```
@@ -69,6 +70,9 @@ class ColdStorageLocationBase(BaseModel):
     critical_threshold_percent: int = Field(
         10, ge=0, le=100, description="Critical at this % free space"
     )
+    is_encrypted: bool = Field(
+        False, description="Whether files in this location should be encrypted"
```

Ruff (0.14.14)

[error] Boolean positional value in function call

(FBT003)